src/Core/Resolvers/MultipleCreateOrderHelper.cs

```diff
@@ -48,7 +50,8 @@ public class MultipleCreateOrderHelper
             ISqlMetadataProvider metadataProvider,
             Dictionary<string, IValueNode?> columnDataInSourceBody,
             IValueNode? targetNodeValue,
-            int nestingLevel)
+            int nestingLevel,
+            bool isMNRelationship = false)
```

Comment:
if a caller knows `isMNRelationship` is true, then they shouldn't call this function to begin with.

Reply:
Discussed over call. Reasoning added to the PR description.